querydsl-codegen/src/main/java/com/querydsl/codegen/EntitySerializer.java

```diff
@@ -620,6 +663,10 @@ public void serialize(EntityType model, SerializerConfig config,
 
     protected void serialize(EntityType model, Property field, Type type, CodeWriter writer,
             String factoryMethod, String... args) throws IOException {
+
+        String constantFieldName = constantFieldName(field.getEscapedName());
+        writer.publicStaticFinal(new ClassType(String.class), constantFieldName, "\"" + field.getName() + "\"");
```

Comment:
Sorry for the radio silence.
Yes, that sounds good.
You can kind of see what places you need to update in [this PR](https://github.com/querydsl/querydsl/pull/1523/files).